app.js

```diff
@@ -4,20 +4,6 @@ import { createStore } from 'redux'
 import * as Components from './components'
```

Comment:
Rule of thumb: only import what you need in a particular file instead of `*`. 

For example:

```javascript
import { GameView } from './components'
```

This will also prevent the need to access objects by using `.`, e.g., `GameView` vs. `Components.GameView`.

Apply this to every relevant place in your PR.